zookeeper.spec

```diff
@@ -45,7 +45,6 @@ mkdir -p $RPM_BUILD_ROOT%{_log_dir}
 mkdir -p $RPM_BUILD_ROOT%{_data_dir}
 mkdir -p $RPM_BUILD_ROOT%{_unitdir}/zookeeper.service.d
 mkdir -p $RPM_BUILD_ROOT%{_conf_dir}/
-install -p -D -m 644 zookeeper-%{version}.jar $RPM_BUILD_ROOT%{_prefix}/zookeeper/
```

Comment:
In 3.4 the main zookeeper jar was in the top-level directory. In 3.5 the main zookeeper jar was moved to the lib/ directory with all the other jars, which is covered by the next line.